Source/Orts.Simulation/Simulation/Activity.cs

```diff
@@ -1334,7 +1334,7 @@ override public Boolean Triggered(Activity activity)
                     {
                         consistTrain = null;
                         consistTrain = matchesConsistNoOrder(ChangeWagonIdList);
-                        triggered = (consistTrain != null ? false : true);
+                        triggered = (consistTrain != null ? true : false);
```

Comment:
Can be simplified to:
```suggestion
                        triggered = consistTrain != null;
```